docs/index.rst

```diff
@@ -172,6 +172,44 @@ uncomment the relevant configuration item(s), and insert your desired value(s).
 
   configuration
 
+
+Caching
+-------
+
+By default Astroquery employs query caching with a timeout of 1 week, although individual services
+may have different settings. The user can clear their cache at any time, as well as suspend cache usage,
```

Comment:
Is it possible for individual services to have a different timeout?

Reply:
Nope